api/EXT_float_blend.json

```diff
@@ -36,7 +36,8 @@
             "version_added": false
           },
           "samsunginternet_android": {
-            "version_added": false
+            "version_added": "11.0",
+            "notes": "Until Samsung Internet 11.0, this extension was unavailable on several Android devices powered by GLES 3.2. See <a href='https://crbug.com/964208'>bug 964208</a>."
```

Comment:
This note seems a little bit odd to me.  Did Samsung Internet merge this patch from Chrome 77 down, so it's available on all GLES 3.2-powered devices?  If so, we should remove this note.  If not, we should adjust the beginning to say "This extension **is** unavailable...".

Reply:
I've looked into this seems, it's not supported. Thanks for pointing it out.